plugin/CactbotEventSource/FFXIVProcessIntl.cs

```diff
@@ -717,7 +719,7 @@ public enum Arcanum : byte {
       }
 
       [FieldOffset(0x04)]
-      private Card _heldCard;
+      private byte _heldCard;
```

Comment:
Should this be `[NonSerialized]`?

Reply:
I'm not sure how `[NonSerialized]` works, but seems you are right.